notebooks/demo.clj

```diff
@@ -1,101 +1,142 @@
 
+The standard way of doing this is something like")
 (def greetings
   (wl/eval
    (w/fn [x] (w/StringJoin "Hello, " x "! This is a Mathematica function's output."))))
-
 (greetings "Stephen")
 
-(md "## Bidirectional translation
-(Somewhat experimental, especially in the wl->clj direction)")
+(k/md "But this can also be done at a lower level, e.g.")
+(def W:Plus
+  (parse/parse-fn 'Plus {:jlink-instance (wolframite.impl.jlink-instance/get)}))
+(W:Plus 1 2 3)
+
+(k/md "## Bidirectional translation (experimental)
+
+Code translation in both directions is more difficult and is still somewhat experimental (especially in the wl->clj direction), but the basics work as expected, e.g.")
 
 (wl/->clj "GridGraph[{5, 5}]")
 (wl/->wl (w/GridGraph [5 5]) {:output-fn str})
 
-(md "## Graphics")
+(k/md "## Graphics
 
-(view (w/GridGraph [5 5]))
+The above code however, within Mathematica, actually produces graphics. Does Wolframite support this?
+
+Yes!
+")
 
-(view (w/GridGraph [5 5])
-      {:folded? true})
+(view (w/GridGraph [5 5]))
 
 (view (w/ChemicalData "Ethanol" "StructureDiagram"))
 
-(md "## More Working Examples")
+(view (w/TextStructure "The cat sat on the mat."))
+
+(k/md "The above graphics were created using the")
+^:kindly/hide-code
+(k/code "view")
+(k/md "function, as required above, and assumes that graphics are to be displayed in a browser.")
 
-(wl/eval (w/GeoNearest (w/Entity "Ocean") w/Here))
+(k/md "## Computational knowledge
 
-(md "TODO: Make this work with `view` as well.")
+Wolfram is also known for its dynamic or 'computational knowledge' engine.
 
-(view (w/TextStructure "The cat sat on the mat."))
+This can be accessed by many functions directly, ")
 
-(md "## Wolfram Alpha")
+(view (w/GeoNearest (w/Entity "Ocean") w/Here))
+(k/md ", or by posting a request to its online platform, Wolfram Alpha.")
 
 (wl/eval (w/WolframAlpha "number of moons of Saturn" "Result"))
 
 (view (w/WolframAlpha "number of moons of Saturn" "Result"))
+(k/md "Here, we've shown a response with and without the view function, for reference.")
+
+(k/md "## Mathematics and beyond
+
+In a nutshell, this is how Wolframite is normally used, but, of course, this barely scratches the surface.
+
+In particular, the flagship product of Wolfram, the one you've probably heard of, is Mathematica. And, as the name suggests, this entire system was built around the performance of abstract calculation and the manipulation of equations, e.g.
+")
+(-> (w/== 'E (w/* 'm (w/Power 'c 2)))
+
+    w/TeXForm
+    w/ToString
+    wl/eval
+    k/tex)
+
+(k/md "originally answered the question 'what is mass?'")
+
+(-> (w/== 'E (w/* 'm (w/Power 'c 2)))
+    (w/Solve 'm)
+    w/First w/First
+
+    w/TeXForm
+    w/ToString
+    wl/eval
+    k/tex)
+
+(k/md "This is where Wolfram, and so Wolframite, really shines. And if you're interested in exploring this further, have a gander at one of our longer tutorials.")
```

Comment:
what's is `gander` ? Could we use a more common word instead?

Reply:
Lol, a little too much British whimsy  I suppose.